vulture_os/toolkit/api_parser/nozomi_probe/nozomi_probe.py

```diff
@@ -151,5 +151,9 @@ def format_log(log):
                     self.frontend.last_api_call = timezone.make_aware(datetime.fromtimestamp(logs[-1]['time']/1000))
                 logger.info(f"[{__parser__}]:execute: Setting last_api_call to {self.frontend.last_api_call}",
                             extra={'frontend': str(self.frontend)})
+            elif len(logs) == 0 and to == since + timedelta(hours=24):
+                self.frontend.last_api_call += timedelta(hours=23, minutes=59)
+                logger.info(f"[{__parser__}]:execute: Setting last_api_call to {self.frontend.last_api_call}",
+                            extra={'frontend': str(self.frontend)})
```

Comment:
```suggestion
            elif to == since + timedelta(hours=24):
                self.frontend.last_api_call += timedelta(hours=23, minutes=59)
                logger.info(f"[{__parser__}]:execute: No log received on time range, next query will be from {self.frontend.last_api_call}",
                            extra={'frontend': str(self.frontend)})
```